### ISSUE TITLE
Track cards that are shuffled into the deck during a game

### ISSUE THREAD

**Issue by extesy** [extesy/DeckTracker#29]
Open question: are there any cards that *replace* cards in the deck instead of adding to it?

Also, this should only work for the player's deck, not for the opponent's deck.

**Comment by ittyzoffeji** [comment]
For TESL the answer is no (at least for now). There are cards however that replace cards in hand.

**Comment by extesy** [comment]
Ok, that's good to know. Cards in hand shouldn't matter because they will be already accounted for.

**Comment by extesy** [comment]
Fixed in `1.0.44` (hopefully, couldn't fully test due to lack of such cards in the collection). Would appreciate someone confirming this.

**Comment by emerald000** [comment]
I'm not sure if that counts, but the Monuments (Amber Monument and cie.) from Eternal transforms into something else when you get 5 power.

**Comment by extesy** [comment]
@emerald000 You mean in the deck list it starts showing up as something else?

**Comment by emerald000** [comment]
You asked if there were cards that replaces other cards in the deck. The Monuments replace themselves no matter where they are once you get to 5 power.

As for the deck list, I saw a couple of revisions ago a "-1 Rhinarc" (the card Amber Monument transforms into) on my decklist. Not sure if that has been fixed since then. But at least the Monuments aren't changing on the decklist when you play your 5th power.
